src/Evolution/DiscontinuousGalerkin/SlopeLimiters/MinmodTci.hpp

```diff
@@ -0,0 +1,47 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <array>
+#include <cstdlib>
+#include <utility>
+
+#include "Evolution/DiscontinuousGalerkin/SlopeLimiters/MinmodType.hpp"
+#include "Utilities/Gsl.hpp"
+
+/// \cond
+class DataVector;
+template <size_t Dim, typename T>
+class DirectionMap;
+template <size_t VolumeDim>
+class Element;
+template <size_t VolumeDim>
+class Mesh;
+/// \endcond
+
+namespace Minmod_detail {
```

Comment:
- This should be in the `SlopeLimiters` namespace, I think. So `SlopeLimiters::Minmod_detail`
- Do we want to move this into a `SlopeLimiters::tci` namespace instead? I'm guessing this will need some more generalization to work more generically, but I'd be in favor of starting the TCI generalization sooner rather than later. In that case we should consider the directory structure too. Maybe `Evo/DG/Tci` (we could spell out TroubledCellIndicator too)? Or we could do `Evo/DG/SlopeLimiters/Tci` (in this case I'd prefer not spelling it out in full since it's expected someone doing limiting knows what TCI is, and if not can look at the docs). This second point is for discussion, not a concrete request :)

Reply:
Re the second point: I agree in general. I suggest waiting until after this PR, perhaps until we're adding a second TCI in the mix and start to have a better idea of how things should look.

Reply:
Talked at SpECTRE hangout. Summary:
- We both agree this should be done but will hold off once the limiter code for WENO is in.